core/main/src/Core/Canvas.ts

```diff
@@ -489,6 +490,22 @@ export class Canvas {
         }
     }
 
+    public drawOrbit(particle: IParticle, orbitOptions: IOrbit): void {
+        if (!this.context) {
+            return;
+        }
+
+        CanvasUtils.drawEllipse(
+            this.context,
+            particle,
+            orbitOptions.color || particle.getFillColor(),
+            orbitOptions.radius || particle.getRadius(),
```

Comment:
Looking at code this radius seems just the value taken directly from the config, but it needs to be updated with retina support like all other radius properties and distances.

Reply:
I've added a method in Retina to fix radius, not sure what to do about distances? Would be great if you could elaborate more on that.
`    
public initParticleOrbit(particle: Particle): void {
        const ratio = this.pixelRatio;

        particle.particlesOptions.orbit.radius = particle.particlesOptions.orbit.radius * ratio;
    }
`

Reply:
A new method is not necessary. You can add it to the `initParticle` method.

I was talking about what could be affected by retina displays, if you are not using distances ignore that part. Checkout how other radiuses work.

Like I said in another comment the `particlesOptions` object should be only read, and the orbit options should be stored inside the `Particle` object like many other values.

I want `options` to be read-only because they are just input values selected by the user, changing them can bring unexpected results when reloading.